docs/en/stock-management/movement.exit.md

```diff
@@ -2,29 +2,221 @@
 
 # Stock Exit
 
-Any outflows of stock from a depot is a stock exit in BHIMA.  The available options are:
+Any outflows of stock from a depot is a stock exit in BHIMA. The exit of stock
+concerns a stock pharmacy as well as a dispensing pharmacies.
 
-1. Exit to Patient *
-2. Exit to a Service *
-3. Transfer to a Depot
-4. Exit to record a Loss
+ - **Movements of stock out of the stock pharmacies or depots** fall into the
+   following cases:
 
-The stock exits marked with an asterisk are _consumptions_ and influence the [average monthly consumption (AMC)](#average-monthly-consumption).  All other types of exits are not considered in the AMC calculation.
+    - **Transfer to Services** [^1] (Wards / Departments): In this case it is
+      assumed that the services are authorized and keep small depot of stock
+      items.  These are typical items such as laxatives, alcohol,
+      anti-bacterial creams, etc.  In short, these are stock items for daily
+      use that are difficult to quantify in advance to be part of a
+      prescription issued for a specific patient.  This category may also
+      include resuscitation and emergency use products that cannot wait for
+      the patient to pay first at the counter before being served at the
+      pharmacy.
 
-Note that stock exits to a patient, service, or recording a loss effectively remove stock items from the enterprise.  The only way to replenish stock after an exit is made is to perform a stock adjustment.
+    - **Transfer to other Depots or dispensing pharmacies**: Normally, patients
+      are served directly at a dispensing pharmacy (or dispensary) when the
+      dispensing pharmacy receives prescription (that have been paid for) at
```

Comment:
Note that BHIMA doesn't insist that a patient pays for their medications.  It only checks to see if the patient has been invoiced.

Reply:
Fixed